django/template/base.py

```diff
@@ -85,12 +85,9 @@
 # (e.g. strings)
 UNKNOWN_SOURCE = '<unknown source>'
 
-# match a variable or block tag and capture the entire tag, including start/end
-# delimiters
-tag_re = (_lazy_re_compile('(%s.*?%s|%s.*?%s|%s.*?%s)' %
-          (re.escape(BLOCK_TAG_START), re.escape(BLOCK_TAG_END),
-           re.escape(VARIABLE_TAG_START), re.escape(VARIABLE_TAG_END),
-           re.escape(COMMENT_TAG_START), re.escape(COMMENT_TAG_END))))
+# match a block, variable, or comment tag and capture the entire tag, including
+# start/end delimiters
```

Comment:
I would include reference to the variable names, since they are now no longer referenced in the string. You could say,

> Match BLOCK_TAG_\*, VARIABLE_TAG_\*, and COMMENT_TAG_\* tags, capturing the
entire tag, including start/end delimiters.